### DISCUSSION TITLE
password icon

### DISCUSSION THREAD

**Discussion by spantheslayer** [tailwindlabs/heroicons#495] [Ideas]
Please make a password icon.

**Comment by ndri** [comment]
I added `password` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=password

**Reply by spantheslayer** [reply]
Yo @ndri this is really cool. Thanks for letting me know. I'll surely make sure to include SideKickIcons to use in my next work.
